jsonrpc-specs/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "portal-network-apis",
+  "version": "0.0.1",
+  "description": "Collection of JSON-RPC APIs provided by Portal Network clients",
+  "main": "index.js",
+  "type": "module",
+  "scripts": {
+    "build": "node scripts/build.js",
+    "lint": "node scripts/build.js && node scripts/validate.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ethereum/portal-network-specs.git"
+  },
+  "author": "Ethereum",
+  "license": "CC0-1.0",
```

Comment:
Does this imply that we need to get this in place in the main repository?  I think we are currently unlicenced.

Reply:
I got this license from [here](https://github.com/ethereum/execution-apis/blob/eab7edd4bcb144cee75233054bcc2016847aeaa9/scripts/build.js#L39). Not sure if it is applicable in our case.